### PR TITLE
[sdg] add geoRegions

### PR DESCRIPTION
**Pull request by chejennifer**
- add geo regions to dropdown

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Thanks!

**Comment by chejennifer** [comment]
thanks for the review!

File: experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/geo_regions.json [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  },
+  {
+    "dcid": "undata-geo/G00152000",
+    "name": "Middle East and North Africa"
+  },
+  {
+    "dcid": "undata-geo/G00162000",
+    "name": "Central and Eastern Europe"
+  },
+  {
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/geo_regions.json:86]: done - added list of regions to filter out

**Review by pradh — COMMENTED** [review]
golden diffs LG

**Pull request merged**